cves/kernel/CVE-2020-12652.yml

```diff
@@ -196,8 +195,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    No sign of any protocols being violated aside from the idea of
+    don't create race conditions.
```

Comment:
This explanation seems a little vague. Could you possibly go into a little more detail on how you came to this conclusion?

Reply:
I added some more detail as to where the conclusion came from, it will be in the next commit.